{{ cookiecutter.repo_name }}/docs/pull_merge_request_templates/GDS.md

```diff
@@ -0,0 +1,32 @@
+# Summary
+
+Add your summary here - keep it brief, to the point, and in plain English.
+
+# Checklists
+
+<!--
+These are DO-CONFIRM checklists; it CONFIRMs that you have DOne each item.
+
+Outstanding actions should be completed before reviewers are assigned; if actions are irrelevant, please try and add a 
+comment stating why.
+
+Incomplete pull/merge requests MAY be blocked until actions are resolved, or closed at the reviewers' discretion.
+-->
+
+### General checks
+
+- [ ] Code runs
+- [ ] Developments are **secure** and **ethical**
+- [ ] You have made proportionate checks that the code works correctly
+- [ ] Test suite passes
+- [ ] Assumptions, and caveats log updated (see `docs/aqa/assumptions_caveats.md`), if necessary
+- [ ] [Minimum usable documentation](http://agilemodeling.com/essays/documentLate.htm) written in the `docs` folder
```

Comment:
useful link

Reply:
not https

Reply:
Leaving this as is, because the `https` version is a bit odd?! If I can find a similar reference in future, I'll swap it out.

Marking this as **Resolved**.